public_suffix_list.dat

```diff
@@ -13345,6 +13345,8 @@ onrender.com
 
 // Repl.it : https://repl.it
 // Submitted by Mason Clayton <mason@repl.it>
```

Comment:
to replace the metadata, add the DNS validation record to the email address for repl.it matching this PR as done for entries.  This would allow us to validate, and then (if needed) update the contact in that header line